junwoo/routes/index.js

```diff
@@ -0,0 +1,15 @@
+//routes/index.js
+
+const express = require("express");
+const router = express.Router();
+
+const userRouter = require("./userRouter");
+router.use("/users", userRouter.router);
+
+const postRouter = require("./postRouter");
+router.use("/posts", postRouter.router);
+
+const likeRouter = require("./likeRouter");
+router.use("/likes", likeRouter.router);
```

Comment:
router 모듈의 미들웨어로 사용되는 각 router 모듈에 미들웨어를 변수에 할당하는 구조와

실제 rotuer 모듈에 미들웨어를 적용하는 코드를 구분해주면서 그 구조를 더욱 명확하게 파악할 수 있을 것 같습니다.

```suggestion
const express = require("express");
const router = express.Router();

const userRouter = require("./userRouter");
const postRouter = require("./postRouter");
const likeRouter = require("./likeRouter");

router.use("/users", userRouter.router);
router.use("/posts", postRouter.router);
router.use("/likes", likeRouter.router);
```